config/knock-knock.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+return [
+    '*' => [
+        'enabled' => false,
+        'enableCpProtection' => false,
+        'loginPath' => 'restricted-access',
+//        'template' => '_knock-knock.twig',
+        'siteSettings' => [],
+        'checkInvalidLogins' => false,
+        'invalidLoginWindowDuration' => '3600',
+        'maxInvalidLogins' => 10,
+        'allowIps' => ['81.82.199.174', '127.0.0.1'],
+        'denyIps' => [],
+        'useRemoteIp' => true,
+        'protectedUrls' => [],
+        'unprotectedUrls' => ['/img/site/logo.png'],
```

Comment:
Url should be changed to "/frontend/img/site/logo.png"

Reply:
Aangepast! kan je nog eens nakijken?